### PR TITLE
Add Sales Channel parameter List all available payment methods

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to add Sales Channel parameter to the List all available payment methods endpoint (EDU-13183).

### CODE REVIEW

File: VTEX - Payments Gateway API.json [hunk 1]
```diff
@@ -2339,6 +2339,17 @@
...
                 "operationId": "AvailablePaymentMethods",
                 "parameters": [
+                    {
+                        "name": "salesChannel",
+                        "in": "query",
+                        "description": "Sales channel identification. This parameter must be filled in if you wish to obtain information on payment methods available in a specific store sales channel.",
```
Comment (Isabella-Veloso) [inline VTEX - Payments Gateway API.json:2345]:
```suggestion
                        "description": "Sales channel ([trade policy](https://help.vtex.com/en/tutorial/how-trade-policies-work--6Xef8PZiFm40kg2STrMkMV) identification. This parameter must be filled in if you wish to obtain information on payment methods available in a specific store sales channel.",
```

**Review by Isabella-Veloso — CHANGES_REQUESTED** [review]
@ricardoaerobr como a nomenclatura para sales channel/trade policy é inconsistente, é uma boa prática nas docs fazer o paralelo.

File: VTEX - Payments Gateway API.json (hunk 1 above)
Comment (ricardoaerobr) [reply VTEX - Payments Gateway API.json:2345]: Boa

**Review by Isabella-Veloso — APPROVED** [review]
LGTM

**Pull request merged**